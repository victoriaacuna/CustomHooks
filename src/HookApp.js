import React from 'react'
import PropTypes from 'prop-types'

const HookApp = props => {
    return (
        <div>
            <h1>Hi</h1>
        </div>
    )
}

HookApp.propTypes = {

}

export default HookApp
