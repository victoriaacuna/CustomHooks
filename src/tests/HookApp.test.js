import React from 'react';
import {shallow} from 'enzyme';
import '@testing-library/jest-dom';
import HookApp from '../HookApp';


describe('Pruebas en HookApp', () => {

    test('should show it correctly', () => {
        const wrapper = shallow(<HookApp/>);
        expect(wrapper).toMatchSnapshot();
    });
    


});