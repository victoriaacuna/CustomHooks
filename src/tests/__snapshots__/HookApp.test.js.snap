// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HookApp should show it correctly 1`] = `
<div>
  <h1>
    Hi
  </h1>
</div>
`;
